assets/app/view/game/part/track_lawson_path.rb

```diff
@@ -37,15 +38,31 @@ def preferred_render_locations
         def render_part
           rotation = 60 * @edge_num
 
-          props = {
-            attrs: {
-              transform: "rotate(#{rotation})",
-              d: 'M 0 87 L 0 0',
-              stroke: @color,
-              'stroke-width': @width,
-              'stroke-dasharray': @dash,
-            },
-          }
+          props =
+            if @terminal == 1
```

Comment:
i think terminal should be a boolean value

you could also just do

props = { ... }

props[attrs].merge(d: ...) if @terminal

Reply:
OK, I'll change to a boolean. It's a pretty drastic change to attrs: so I don't know if the merge would be as pretty.

Reply:
But I will defer to your judgement.